src/mugs.js

```diff
@@ -293,7 +293,45 @@ define([
             rawControlXML: {
                 presence: 'optional',
                 lstring: 'Raw XML'
-            }
+            },
+            dataParent: {
+                lstring: 'Data Parent',
+                visibility: function(mug) {
+                    function recFunc(mug) {
+                        if (!mug) {
+                            return true;
+                        } else if (mug.__className === 'Repeat') {
+                            return false;
+                        }
+                        return recFunc(mug.form.controlTree.getNodeFromMug(mug).parent.value);
```

Comment:
I tested this with hidden values. That code doesn't get to this function because dataParent is not in the spec. Should I also test it with any other mugs (didn't see any others when I first searched through)?